src/current/v24.1/online-schema-changes.md

```diff
@@ -81,7 +81,7 @@ The following statements use the declarative schema changer by default:
 - [`ALTER TABLE ... VALIDATE CONSTRAINT`]({% link {{ page.version.version }}/alter-table.md %}#drop-constraint)
 - [`ALTER TABLE ... DROP CONSTRAINT`]({% link {{ page.version.version }}/alter-table.md %}#validate-constraint)
 
-Until all schema change statements are moved to use the declarative schema changer you can enable and disable the declarative schema changer for supported statements using the `sql.defaults.use_declarative_schema_changer` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) and the `use_declarative_schema_changer` [session variable]({% link {{ page.version.version }}/set-vars.md %}).
+Until all schema change statements are moved to use the declarative schema changer you can enable and disable the declarative schema changer for supported statements using the `sql.defaults.use_declarative_schema_changer` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#sql-defaults-use_declarative_schema_changer) and the `use_declarative_schema_changer` [session variable]({% link {{ page.version.version }}/set-vars.md %}#use_declarative_schema_changer).
```

Comment:
to fix deploy preview error: `hash does not exist --- docs/v24.1/online-schema-changes.html --> /docs/v24.1/cluster-settings.html#sql-defaults-use_declarative_schema_changer`

```suggestion
Until all schema change statements are moved to use the declarative schema changer you can enable and disable the declarative schema changer for supported statements using the `sql.defaults.use_declarative_schema_changer` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-defaults-use_declarative_schema_changer) and the `use_declarative_schema_changer` [session variable]({% link {{ page.version.version }}/set-vars.md %}#use_declarative_schema_changer).
```

Reply:
I will review again once the deploy preview passes.

Reply:
thanks, should be fixed with latest push